### PR TITLE
[VDR ]add ohif brief overview doc in the readme

### LINKED ISSUE #65: Need to add a section about the OHIF view demo application in the documentation
- Brief description about the application
- UI description (which is something different and new for MONAI service users, do not need to explain the basic OHIF viewer design/UI components)
- Tutorial/Sample workflows:

### PR DESCRIPTION
**Pull request by tangy5**
Fix #65 , #66

### CODE REVIEW

**Comment by mingxin-zheng** [comment]
Hi @tangy5 , can we change the merge target to @zephyrie 's `integrated_doc` branch?

File: README.md [hunk 1]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+   - [OHIF Applications and UI Demo](#basic-ohif-instruction)
+   - [Interactive Annotation](#interactive-annotation)
+   - [Notebook: Annotation and OHIF Overview](https://github.com/NVIDIA/monai-cloud-api/blob/main/notebooks/Annotation%20and%20Continuous%20Learning%20Overview.ipynb)
+
+
+## How to Get Started
 
 To begin using the NVIDIA MONAI Cloud APIs, you can sign up for early access by filling out this survey: [NVIDIA MONAI Cloud API Early Access](https://developer.nvidia.com/nvidia-monai-cloud-api-early-access-program/join). 
+
+## Getting Started with OHIF Plugin
```
Comment (mingxin-zheng) [inline README.md:34]: If we folllow the style @zephyrie 's `integrated_doc` branch, I would suggest to move the contents of OHIF and DICOMweb to two markdown files.

File: README.md [hunk 2]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+
+MONAI Service provide ready-to-use playgroundw with API server and applications for users.
```
Comment (mingxin-zheng) [inline README.md:36]: typo: playgroundw

Comment (mingxin-zheng) [reply README.md:36]: The whole sentence can be
```
The MONAI Service offers a ready-to-use environment featuring an API server and applications for users. If you wish to construct and customize the OHIF viewer along with the MONAI Service plugin, please consult the [setup page](https://github.com/NVIDIA/monai-cloud-api/tree/main/plugins/ohif).
```

File: README.md [hunk 3]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
 
 To begin using the NVIDIA MONAI Cloud APIs, you can sign up for early access by filling out this survey: [NVIDIA MONAI Cloud API Early Access](https://developer.nvidia.com/nvidia-monai-cloud-api-early-access-program/join). 
+
+## Getting Started with OHIF Plugin
+
+MONAI Service provide ready-to-use playgroundw with API server and applications for users. 
+If you are going to build and custimize OHIF viewer and MONAI Service plugin. Please refer to the [setup page](https://github.com/NVIDIA/monai-cloud-api/tree/main/plugins/ohif)
+
+### Dicomweb Overview
+Dataset storage can be variant, in the open-sourced setup. We recommend Orthanc for hosting dicom format medical images.
```
Comment (mingxin-zheng) [inline README.md:40]: The storage of datasets can vary; in our open-source setup, we recommend using Orthanc for hosting **DICOM** format medical images.

File: README.md [hunk 4]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+For customized setup for Orthanc, 
+
+**Steps:**
+
+```bash
+# Install Orthanc and DICOMweb plugin
+sudo apt-get install orthanc orthanc-dicomweb -y
+
+# Install Plastimatch
+sudo apt-get install plastimatch -y
```
Comment (mingxin-zheng) [inline README.md:59]: Can these two be merged into one-line?
```
sudo apt-get install orthanc orthanc-dicomweb plastimatch -y
```

File: README.md [hunk 5]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+To create dataset, then you ready to create a reference object to that dataset by utilizing DICOMWeb. 
+DICOMWeb is a modern web standard for accessing DICOM data. 
+By connecting NVIDIA MONAI Cloud APIs to a DICOMWeb endpoint, 
+you can seamlessly integrate to your data into many modern viewers.
+
+In MONAI Service playground, it will automatically deploy Orthanc dicom-web server. 
+For customized setup for Orthanc, 
+
+**Steps:**
+
```
Comment (mingxin-zheng) [inline README.md:53]: Add 
```
1. Install Orthanc and DICOMweb plugin
```

File: README.md [hunk 6]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+sudo wget https://lsb.orthanc-server.com/orthanc/1.9.7/libServeFolders.so --output-document /usr/share/orthanc/plugins/libServeFolders.so
+sudo wget https://lsb.orthanc-server.com/orthanc/1.9.7/libModalityWorklists.so --output-document /usr/share/orthanc/plugins/libModalityWorklists.so
+sudo wget https://lsb.orthanc-server.com/plugin-dicom-web/1.6/libOrthancDicomWeb.so --output-document /usr/share/orthanc/plugins/libOrthancDicomWeb.so
+
+sudo service orthanc restart
+```
+
+```
+dicom_web_endpoint = ...
```
Comment (mingxin-zheng) [inline README.md:77]: Where will these variables be consumed? MONAI cloud api notebooks?

File: README.md [hunk 7]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+"RegisteredUsers" : {
+  "user": "pass"
+}
+```
+
+### Basic OHIF Instruction
+
+
+#### 1. **Study List**
+Open OHIF viewer, the index page is a study list. The subjects listed are prepared in dicom-web storage.
```
Comment (mingxin-zheng) [inline README.md:100]: Incomplete sentence:
```
Open OHIF viewer, the index page is a study list
```

Comment (mingxin-zheng) [reply README.md:100]: **DICOMweb**

File: README.md [hunk 8]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+#### 3. **Selected MONAI Service Plugin**
+
+Click the MONAI Service plugin button on the right panel after the tri-planer views are up. 
+The MONAI Service planel will show on the right, the plugin will automatically connect
+to the running MONAI Service server and to load executed models and created datasets. 
+![allclass](docs/doc_imgs/viewerpage.jpg)
+
+
+
+#### 4. **Run Inferencing Using Selected Method**
```
Comment (mingxin-zheng) [inline README.md:123]: After this MR is merged, we can remove the duplicated contents in the notebook: https://github.com/NVIDIA/monai-cloud-api/blob/main_ea4/notebooks/Annotation%20and%20Continuous%20Learning%20Overview.ipynb

File: README.md [hunk 9]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+Click **Run** after the click the points for AI based inference.
+
+![study list](docs/doc_imgs/inference_point_res.png)
+
+User can get the real-time prompt response for the point edits.
+
+**Class Prompts**:
+
+The class prompts is used when inference and annotating specific anatomies, VISTA model supprts more
+ than 100 organs, bones, muscles, etc. Users can selecte one or multiple targets for AI inference.
```
Comment (mingxin-zheng) [inline README.md:195]: selecte

File: README.md [hunk 10]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+### Interactive Annotation
+MONAI Service supports multiple applications such as VISTA and DeepEdit, VISTA model contains two interactive mode:
+point prompt and class prompt. DeepEdit has only the point prompt interactive annotation. 
+
+#### **Interactive Mode: Point Prompt**
+
+Use mouse and click to place the control points within medical images. 
+In OHIF MONAI Service plugin, user can use point prompts by clicking the
+**Point Prompts**:
+You can see availabel organs in the panel, select one anatomy for point
```
Comment (mingxin-zheng) [inline README.md:180]: availabel

File: README.md [hunk 11]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+
+![Annotate](docs/doc_imgs/annotate.png)
+
+#### 6. **Save and Notify the Server**
+
+Once you're satisfied with your annotations, the first step is to save the annotated image, ensuring that your work is captured. This will write back the image using the DICOMWeb protocal back to your datastore.
+
+![Save Label](docs/doc_imgs/savelabel.png)
+
+Next, notify the server that an image has been annotated. This step is crucial for continuous learning. The system will take note of the new annotations and after the indicated number of annotated images it will use them to improve the model over time.
```
Comment (mingxin-zheng) [inline README.md:159]: continuous learning -> continual learning

File: README.md [hunk 12]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+With the initial mask in place, you might notice areas that require manual tweaking. 
+Use the provided annotation tools to:
+
+- Refine boundaries
+- Add or remove regions
+
+This step ensures that your annotations are as accurate as possible.
+
+**Steps**
+1. Click the Segmentation button.
```
Comment (mingxin-zheng) [inline README.md:146]: ~~button~~

File: README.md [hunk 13]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+2. Select a class of segmentation that needs to be updated.
```
Comment (mingxin-zheng) [inline README.md:147]: that needs update

File: README.md [hunk 14]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+
+#### 5. **Annotate / Refine Annotations use side panel **
+
+With the initial mask in place, you might notice areas that require manual tweaking. 
+Use the provided annotation tools to:
+
+- Refine boundaries
+- Add or remove regions
+
+This step ensures that your annotations are as accurate as possible.
```
Comment (mingxin-zheng) [inline README.md:143]: This step is crucial to ensure the highest possible accuracy of your annotations.

File: README.md [hunk 15]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+
+Click **Run** after the click the points for AI based inference.
+
+![study list](docs/doc_imgs/inference_point_res.png)
+
+User can get the real-time prompt response for the point edits.
+
+**Class Prompts**:
+
+The class prompts is used when inference and annotating specific anatomies, VISTA model supprts more
```
Comment (mingxin-zheng) [inline README.md:194]: "The class prompt is" or "The class prompts are"

Comment (mingxin-zheng) [reply README.md:194]: supprts

File: README.md [hunk 16]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+
+Select the anatomy by checking the checkbox table:
+
+![study list](docs/doc_imgs/inference_class_prompts.png)
+
+Click **Run** after the classes checkbox for AI based inference.
+
+![study list](docs/doc_imgs/inference_class_prompts_res.png)
+
+User can get the real-time prompt response for the class edits.
```
Comment (mingxin-zheng) [inline README.md:205]: Would be good to add a small summary or conclusion after this line.

File: README.md (hunk 1 above)
Comment (zephyrie) [reply README.md:34]: Yeah, we should break this down so people can find the relevant section and we should not have this in the README but in the docs section for How-To guides. 
"DICOMWeb Setup" and "Using OHIF and MONAI Cloud API Plugin", we can also move the instructions from the OHIF plugin setup directory to another docs page "Setting up OHIF and MONAI Cloud API Plugin"

Comment (zephyrie) [reply README.md:34]: Also, can we update the plugin name / any other displayed name to "NVIDIA MONAI Cloud API"? We should stop using MONAI Service for any naming moving forward.

File: README.md (hunk 4 above)
Comment (zephyrie) [reply README.md:59]: As far as I can tell, we don't need plastimatch for orthanc, but for the conversion of nifti to dicom?  We can leave it as a separate code block, but we should give me information on why we need it, give an example of using it to convert, and also specify the installation is optional and only required for the conversion of files.

File: README.md (hunk 8 above)
Comment (zephyrie) [reply README.md:123]: Yeah, we should move as much as possible that's duplicate or unnecessary for the actual flow of the notebook to docs.

File: README.md [hunk 17]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
...
+
+Once you're satisfied with your annotations, the first step is to save the annotated image, ensuring that your work is captured. This will write back the image using the DICOMWeb protocal back to your datastore.
+
+![Save Label](docs/doc_imgs/savelabel.png)
+
+Next, notify the server that an image has been annotated. This step is crucial for continuous learning. The system will take note of the new annotations and after the indicated number of annotated images it will use them to improve the model over time.
+
+![Notify](docs/doc_imgs/notify.png)
+
+The associated API call run when you click the `Notify Server` button is below:
```
Comment (zephyrie) [inline README.md:163]: Remove, we don't need to inform people during this guide what the corresponding API call is.  We should define a Viewer Integration guide that talks about the dataset API calls and how they should be used.

File: README.md [hunk 18]
```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
    - A robust foundation model, trained on 117 classes encompassing various organs and tumors.
    - The model's training and validation involved CT images from over 4,000 patients, ensuring its high accuracy and reliability for medical image analysis.
 
-### How to Get Started
+4. Dicon-web Setup and Dataset Configuration:
```
Comment (mingxin-zheng) [inline README.md:18]: Dicom

**Comment by mingxin-zheng** [comment]
Closing as the doc is available elsewhere.

**Pull request closed without merging**